style.css

```diff
@@ -365,6 +371,93 @@ hr {
   border-radius: 8px;
 }
 
+/* Pop-up  */
+.pop-up-window {
+  transform: scale(0);
+  transition: 200ms ease-in-out;
+  background: #fff;
+  position: fixed;
+  border-radius: 12px;
+  z-index: 10;
+  margin: 5px;
+  padding: 6px;
+  margin-top: 18px;
+  margin-right: 19px;
+  top: 0;
+  bottom: 0;
+}
+
+.pop-up-window.active {
+  transform: scale(1);
+  overflow: scroll;
+}
```

Comment:
Great job team 🤝

- [ ] Just like the previous reviewer pointed out, your popup design (image) appears blurred and stretched in full-screen view. You may need to fix it so that it replicates the design provided in the Figma template.
- [ ] Also as the previous reviewer pointed out, your buttons are not well aligned as the design hits the full-screen view. You may kindly fix that as well.
- Your design
![Your popup](https://user-images.githubusercontent.com/63917725/180220771-f3cefc8c-0f2a-45c9-a58e-ab78f405c7eb.PNG)

- Figma design
![Figma popup](https://user-images.githubusercontent.com/63917725/180220846-25187144-4e1f-4eb0-b45f-474f693d8f10.PNG)